### ISSUE TITLE
[BUG]  `backdrop-filter` CSS property flagged as invalid

### ISSUE THREAD

**Issue by yehuda100** [Cyb3r-Jak3/html5validator-action#46]
**Describe the bug**  
When using the CSS property `backdrop-filter` in a stylesheet, the validator incorrectly flags it as non-existent:  
> `"backdrop-filter": Property "backdrop-filter" doesn't exist.`  
This property is part of the standard CSS specification and is supported by all major modern browsers.

**To Reproduce**  

1. Go to the CSS editor or upload a stylesheet containing `backdrop-filter`.
2. Use a rule like:
```css
.btn {
  backdrop-filter: blur(10px);
}
```
3. Run the validation.
4. Observe the error message stating the property doesn't exist.

### What is the current *bug* behavior  
The validator incorrectly reports `backdrop-filter` as an invalid or unknown CSS property.

**Expected behavior**  
The validator should recognize `backdrop-filter` as a valid CSS property and not flag it as an error.  
It is part of the official CSS spec and widely supported across modern browsers (Chrome, Edge, Safari).

**Screenshots**  
If applicable, include a screenshot of the error message from the validator interface.

**Additional context**  
- The issue may stem from outdated CSS property definitions in the validator’s ruleset.
- A workaround is to use `-webkit-backdrop-filter`, but this should no longer be necessary.
- Reference: MDN Web Docs – backdrop-filter: https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter

**Comment by Cyb3r-Jak3** [comment]
My guess this is related to an old version of https://github.com/validator/validator in svenkreiss/html5validator.

**Comment by Cyb3r-Jak3** [comment]
@yehuda100 can you try with version 8.0.0?
